compiler/frontend/src/org/jetbrains/kotlin/diagnostics/PositioningStrategies.kt

```diff
@@ -745,19 +743,29 @@ object PositioningStrategies {
         }
     }
 
-    val REFERENCE_BY_QUALIFIED: PositioningStrategy<PsiElement> = object : PositioningStrategy<PsiElement>() {
+    val REFERENCE_BY_QUALIFIED: PositioningStrategy<PsiElement> = FindReferencePositioningStrategy(false)
+    val REFERENCED_NAME_BY_QUALIFIED: PositioningStrategy<PsiElement> = FindReferencePositioningStrategy(true)
+
+    class FindReferencePositioningStrategy(val locateReferencedName: Boolean) : PositioningStrategy<PsiElement>() {
```

Comment:
Add a similar doc comment?

```
/**
 * @param locateReferencedName whether to remove any nested parentheses while locating the reference element.
 */
```

Reply:
Hmm, it's added in https://github.com/JetBrains/kotlin/pull/4186/commits/89603e7dee54cbed0dd90df11ea44c017238f367#diff-9823954b86b2503568b6c9e344d16a76b7c855ea6e56b69957d79541f4d3c946R479